fastlane/lib/fastlane/plugins/plugin_manager.rb

```diff
@@ -0,0 +1,338 @@
+module Fastlane
+  class PluginManager
+    require "bundler"
+
+    PLUGINFILE_NAME = "Pluginfile".freeze
+    DEFAULT_GEMFILE_PATH = "Gemfile".freeze
+    GEMFILE_SOURCE_LINE = "source \"https://rubygems.org\"\n"
```

Comment:
Can we modify this string to also include the "Ensure this file is checked in to source control" line so that brand new `Gemfile`s